.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -1807,6 +1958,92 @@ stages:
             command: "CheckBuildLogsForErrors"
             apiKey: $(DD_LOGGER_DD_API_KEY)
 
+- stage: integration_tests_arm64_debugger
```

Comment:
Sorry, do you mean `coverage` stage at `ultimate-pipeline.yml`?
If yes, it's kind of problematic, since `debugger` will run conditionally, based on `ifDebuggerChanged` variable.
Do you mean something else?

Reply:
Yes, that's what I meant, but as discussed offline, that could be problematic for several reasons:
- The debugger integration tests only run if there are changes in the debugger. This would prevent the coverage stage from running if the debugger tests don't run.
- If we updated the coverage stage to _always_ run, then the coverage numbers would jump around, depending on whether the debugger has run.

As a workaround, we should:
- Introduce a new `coverage_debugger` stage
- In the existing `coverage` stage _don't_ download the coverage reports from the debugger stages
- In the new `coverage_debugger` stage _only_ download the coverage reports from the debugger stages
- Post two coverage report comments to the PR, one for each coverage stage.

This isn't ideal, as neither report gives a "true" value of the coverage, but it's better than nothing, and we're more interested in trends than in raw numbers.

This is sufficiently annoying that we should do this in a new PR 😄

Reply:
@OmerRaviv We need to add this task to our backlog and implement in near future.

Reply:
Agreed 👍  Please go ahead and create that task.

Reply:
Done.
Resolving for now.